pyinfra/facts/windows.py

```diff
@@ -58,27 +59,48 @@ def process(output):
         return new_output
 
 
+def _format_windows_for_key(subject, primary_key, output):
+    '''Format the windows powershell output that uses 'Format-Line'
+       into a dict of dicts
+    '''
+    primary_key = primary_key.strip()
+    lines = {}
+    one_item = {}
+    key_value = ''
+    for line in output:
+        # split line on ':'
+        line_data = line.split(':')
+        if len(line_data) > 1:
+            # we have a data line
+            this_key = line_data[0].strip()
+            this_data = line_data[1].strip()
+            if len(line_data) > 2:
+                # there was a ':' in the data, so reconstitute the value
+                this_data = ':'.join(line_data[1:]).strip()
+            if this_key != primary_key:
+                one_item.update({this_key: this_data})
+            else:
+                key_value = this_data
+        else:
+            # we probably came across a blank line, write the line of data
+            if one_item:
+                lines[key_value] = one_item
+                one_item = {}
+                key_value = ''
+    return {subject: {primary_key: lines}}
```

Comment:
We shouldn't wrap everything here by `subject` rather just have the fact data.

Reply:
Ok. I agree.